lib/dal/src/socket/output.rs

```diff
@@ -495,6 +495,8 @@ impl From<OutputSocket> for frontend_types::OutputSocket {
         Self {
             id: value.id.into(),
             name: value.name,
+            //default to false, but figure out how to do this better
+            eligible_to_recieve_data: false,
```

Comment:
typo on "recieve" 😅

Reply:
"I before E except after C" --- EXCEPT FOR SCIENCE!!